README.md

```diff
@@ -4,69 +4,76 @@
+* that is part of a library, such as oneMKL  
+
+The `dpctl` library is built upon the [SYCL*
+standard](https://www.khronos.org/sycl/). It also implements Python
 bindings for a subset of the standard [runtime
-classes](https://www.khronos.org/registry/SYCL/specs/sycl-2020/html/sycl-2020.html#_sycl_runtime_classes)
-that allow users to query platforms, discover and represent devices
-and sub-devices, and construct contexts and queues.  In addition,
-`dpctl` features classes for [SYCL Unified Shared Memory
+classes](https://www.khronos.org/registry/SYCL/specs/sycl-2020/html/sycl-2020.html#_sycl_runtime_classes).
+Such classes allow users to:
+* query platforms
+* discover and represent devices and sub-devices
+* construct contexts and queues 
+
+`dpctl` features classes for [SYCL* Unified Shared Memory
 (USM)](https://link.springer.com/chapter/10.1007/978-1-4842-5574-2_6)
 management and implements a tensor [array
 API](https://data-apis.org/array-api/latest/).
 
-The library also assists authors of Python native extensions written
-in C, Cython, or pybind11 to access `dpctl` objects representing SYCL
+The library helps authors of Python native extensions written
+in C*, Cython*, or pybind11* to access `dpctl` objects representing SYCL*
 devices, queues, memory, and tensors.
 
-`Dpctl` is the core part of a larger family of [data-parallel Python
-libraries and
-tools](https://www.intel.com/content/www/us/en/developer/tools/oneapi/distribution-for-python.html)
-to program XPUs. The library is available via
-[conda](https://anaconda.org/intel/dpctl) and
-[pip](https://pypi.org/project/dpctl/).  It is included in the [Intel(R)
+`Dpctl` is the core part of a larger family of [data-parallel Python*
+libraries and tools](https://www.intel.com/content/www/us/en/developer/tools/oneapi/distribution-for-python.html)
+to program on XPUs. 
+
+
+
+# Installing
+
+You can install the library with [conda*](https://anaconda.org/intel/dpctl) and
+[pip*](https://pypi.org/project/dpctl/). It is also available in the [Intel(R)
 Distribution for
 Python*](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/distribution-for-python.html)
 (IDP).
 
-Installing
-==========
+## Inte(R) oneAPI
 
-From Intel oneAPI
------------------
+You can find the most recent release of `dpctl` every quarter as part of the Intel(R) oneAPI releases. 
 
-`dpctl` is packaged as part of the quarterly Intel oneAPI releases. To
-get the library from the latest oneAPI release please follow the
-instructions from Intel's [oneAPI installation
+To get the library from the latest oneAPI release, follow the
+instructions from Intel(R) [oneAPI installation
 guide](https://www.intel.com/content/www/us/en/developer/articles/guide/installation-guide-for-oneapi-toolkits.html).
-Note that you will need to install the Intel BaseKit toolkit to get
-IDP and `dpctl`.
 
-From Conda
-----------
+> **NOTE:** You need to install the Intel(R) oneAPI Base Toolkit to get
```

Comment:
Should we change it to:
```suggestion
> **NOTE:** You need to install the Intel(R) AI Analytics Toolkit (AI Kit) to get
```

Reply:
We are in the process of transition from BaseKit to AI Kit, but that transition will only become public in 2023. Since AI kit build on BaseKit changing it now will not be misleading.